rousseau_vote/l10n/it.json

```diff
@@ -59,5 +59,25 @@
     "drawer-blog": "Il Blog Delle Stelle",
     "drawer-feedback": "Invia un feedback",
     "drawer-privacy": "Privacy",
-    "drawer-logout": "Logout"
+    "drawer-logout": "Logout",
+
+    
+    "special_mentions": "Menzioni speciali",
+    "special_mentions_tooltip": "Il candidato presenta nel suo Curriculum Vitae menzioni speciali o riconoscimenti riconducibili ad ambiti personali o professionali, o riconoscimenti pubblici rilevanti",
+    "high_specialization": "Alta specializzazione",
+    "high_specialization_tooltip": "Il candidato presenta nel suo Curriculum Vitae esperienze lavorative di eccellenza o un alto livello di specializzazione professionale",
+    "sharing_proposer": "Proponente Sharing",
+    "activism_organizer":"Organizzatore Activism",
+    "villaggio_rousseau_volunteer":"Volontario Villaggio Rousseau",
+    "user_lex_proposer": "Proponente lex iscritti",
+    "villaggio_rousseau_participant":"Partecipante Villaggio Rousseau",
+    "elarning_student": "Studente Elearning",
+    "openday_participant": "Partecipante Openday",
+    "list_representative": "Rappresentante di lista",
+    "graduate":"Laureato",
+    "italia_cinque_stelle_volunteer":"Volontario Italia 5 Stelle",
```

Comment:
It might look trivial but style (spacing in this case) is really important. We should keep an homogeneous style with a single space. I'm thinking about adding a lint check for each commit.

Reply:
ok